storybookjs/storybook
# Add rebass example to docs

Issue: N/A

## What I did

Added rebass example

## How to test

```
cd docs
yarn dev
```

1. click on "see more examples" link
2. view Rebass example
